utopian-io/v2.utopian.io
# Add the possibility to have more than one project owner

### Verify that

NB: I'll add the tests in another PR because I need to build a complete set with the users etc...
NB: the logged in user can't remove himself from the project. I've made this choice for now, we'll see if we want another behavior

### **Test plan**

_Add another owner_

* Create a project and add one or more other owners

**Expected result:** You should see in the database all the owners added plus the user that created the project

_Playing around with the component_

* Try to break the component and the API!

**Expected result:** You shouldn't be able to
